packages/wonder-blocks-dropdown/components/multi-select.test.js

```diff
@@ -32,6 +35,7 @@ describe("MultiSelect", () => {
                 <OptionItem label="item 1" value="1" />
                 <OptionItem label="item 2" value="2" />
                 <OptionItem label="item 3" value="3" />
+                {false && <OptionItem label="item 4" value="4" />}
```

Comment:
You might not need this line, unless I'm missing something.

Reply:
Oh this was intentional to test the "falsey" items shouldn't get considered in total item count.

Reply:
Can you add a comment to that effect so that maintainers understand why this is here?